README.md

```diff
@@ -81,49 +81,75 @@ import { UISizePlugin, ControlsPlugin, Container } from 'springroll-container';
+      //LayersPlugin controls the progressive removal of distracting game layers. I.e. the higher the slider the more layers should be hidden from player view.
+      new LayersPlugin({
+        //Expects an HTML Input Element of type="range"
+        layersSlider: '#layers-slider-selector' // goes from 0.00 to 1.00 (two points of precision)
+      }),
+      //DifficultyPlugin controls the difficulty/speed of the game.
+      new DifficultyPlugin({
+        //Expects an HTML Input Element of type="range"
+        difficultySlider: '#difficulty-slider-selector',
+        difficulty: 0.5 // goes from 0.1 to 1.0 (one points of precision). Default = 0.5
       }),
     ]
   });
 	container.openPath('game.html');
 ```
-LayersPlugin
+The following plugins require an extra bit of configuration from the game application to function correctly:
+
+HUDPlugin
 ```javascript
-import { LayersPlugin, Container } from 'springroll-container';
+import { HUDPlugin, Container } from 'springroll-container';
 
   const container = new springroll.Container({
     iframeSelector: "#game",
     plugins: [
-      //LayersPlugin controls the progressive removal of distracting game layers. I.e. the higher the slider the more layers should be hidden from player view.
-      new LayersPlugin({
-        //Expects an HTML Input Element of type="range"
-        layersSlider: '#layers-slider-selector' // goes from 0.00 to 1.00 (two points of precision)
+      //HUDPlugin expects a button element/selector string
+      new HUDPlugin({
+        hudSelectorButton: '#hud-position-button-selector' //toggles through the available HUD positions reported by the game
       }),
     ]
   });
 	container.openPath('game.html');
 ```
-HUDPlugin *(see note below)
+*The HUDPlugin requests the supported positions directly from the game itself and builds out an internal array of positions dynamically
+e.g. if the game supports ['top', 'bottom'] then the Plugin will toggle between those two options whenever the button is clicked. See [the SpringRoll Application Class docs](https://github.com/SpringRoll/SpringRoll/tree/v2/src#handling-state-change) for more information on the request format.
+
+ControlsPlugin
 ```javascript
-import { HUDPlugin, Container } from 'springroll-container';
+import { ControlsPlugin, Container } from 'springroll-container';
 
   const container = new springroll.Container({
     iframeSelector: "#game",
     plugins: [
-      //HUDPlugin expects a container/wrapper element.
-      new HUDPlugin({
-        positionsContainer: '#a-div-or-other-wrapper' //any element that can accept radio buttons inside it is acceptable
+      //ControlsPlugin also accepts an [optional] initial value for its control sensitivity
+      new ControlsPlugin({
+        sensitivitySlider: '#sensitivity-slider-selector',
+        sensitivity: 0.5, //control sensitivity goes from 0.1 to 1.0 (1 point of precision) default = 0.5
```

Comment:
Similarly, let's make this field called `default` to be consistent.

Reply:
switched to defaultPointerSize and defaultButtonSize